cedar-policy-validator/src/rbac.rs

```diff
@@ -79,7 +78,6 @@ impl Validator {
         let known_action_ids = self
             .schema
             .known_action_ids()
-            .map(ToString::to_string)
             .collect::<Vec<_>>();
```

Comment:
Might be able to get rid of this `collect` by using `&Schema` as the parameter type for `unrecognized_action_id_help`

Reply:
Ok, we're now passing `&Schema`